src/hypertrace/agent/init/__init__.py

```diff
@@ -78,11 +79,21 @@ def init_trace_provider(self) -> None:
 
     def init_propagation(self) -> None:
         '''Initialize requested context propagation protocols.'''
+        propagator_list = []
         for prop_format in self._config.agent_config.propagation_formats:
+            if prop_format == config_pb2.PropagationFormat.TRACECONTEXT:
+                # TRACECONTEXT is the default context propagation
+                from opentelemetry.propagate import get_global_textmap # pylint: disable=C0415
+                propagator_list += get_global_textmap()._propagators # pylint: disable=W0212
```

Comment:
Quick question, why do we need to look at global textmap? If agent is being initialized we should just override it. Also I don't think we should be accessing directly protected methods.

Reply:
It wasn't immediately obvious how to build that object from scratch.

Reply:
I see. Can we remove it?

Reply:
We have to come up with that existing TRACECONTEXT trace propagator somehow in order to support using both protocols at the same time. Are we missing something regarding the requirement?

Reply:
Can't we do the same as in https://github.com/hypertrace/pythonagent/pull/188/files#diff-736afe97be2f3511e940209df9eb9987ec166fead587bcd9cd450f69523cf328R92? I mean, first because it is explicit and second because we truly don't know what is inside get_global_textmap()._propagators. We expect it to include TraceContext but it does not have to be like that.